cloaked.js

```diff
@@ -77,20 +79,22 @@ function updateListView() {
         $('#listView').append(html);
     }
 }
+*/
 
 // Handles searching for spoilers
 function searchForSpoilers() {
     if (spoilerList["spoilerItem"] != null) {
         var searchString = '';
         spoilerList["spoilerItem"].forEach(function (item) {
-            searchString = searchString + "*:icontains('" + item + "'), ";
+
+        searchString = searchString + "p:icontains('" + item + "'), h1:icontains('" + item + "'), h2:icontains('" + item + "'),  h3:icontains('" + item + "'), h4:icontains('" + item + "'),  h5:icontains('" + item + "'), h6:icontains('" + item + "'), li:icontains('" + item + "'), ";
         });
         searchString = searchString.substring(0, searchString.length - 2);
-        $(searchString).parent().css('-webkit-filter', 'blur(5px)');
+        $(searchString).parent(":not('body')", ":not('head')").css('-webkit-filter', 'blur(5px)');
```

Comment:
To clarify this logic, what you're saying is here is that we're explicitly excluding the 'body' and 'head' tags, right?

Reply:
Correct, how this only takes checks when it is the `parent` that is a `body` or `head`, as it is passed in as an argument to the parent method.